backend/app/GH_API_handling.py

```diff
@@ -0,0 +1,70 @@
+from github import Github
+from sqlalchemy.exc import IntegrityError
+from . import db
+from app.models import GithubUserInfo
+from datetime import datetime
+
+
+"""
+    To do:
+        add file 'GH_access_token.txt' to 'backend/app' with github access token 
+        token must be without '' and ""
+"""
+
+with open("./app/GH_access_token.txt") as f:
+    GH_access_token = f.read().strip()
+
+
+def search_for_user(username=None):
```

Comment:
I'd format this file so that there are fewer empty lines and it's easier to read